### ISSUE TITLE
Choose whether to use GPT-4 fast or slow (i.e. as part of quota or not)

### ISSUE THREAD

**Issue by CesarERamosMedina** [getcursor/cursor#831]
**Is your feature request related to a problem? Please describe.**
Not a problem per se. I would like to be able to choose when to run my GPT-4 calls in non-fast mode. There are times when I absolutely need to run GPT-4 fast, and I would hate to have to do it when I'm out of quota

**Describe the solution you'd like**
Being able to either set a default mode for GPT-4, or being able to change whether to run on slow or fast mode

**Comment by mbtre** [comment]
Yes, looking forward for this as well. Given its current marketing, I assumed you'd have control over this feature. There is actually a button "fast mode":

<img width="644" alt="image" src="https://github.com/getcursor/cursor/assets/128634931/32a77d87-9633-4127-9799-dcedde242874">


However cursor seems to consume fast requests without that "fast mode" anyway...

**Comment by angelmic** [comment]
+1, not only for GPT-4 but also for all available models.
